README.md

```diff
@@ -1,10 +1,41 @@
-# Infinite Industries 
+# Infinite Industries
+<img src="./docs/doc-splash.png">
 
 ## Mission Statement
+<p>Infinite Industries makes experimental contemporary culture more accessible to everyone. Our digital platform and open-source tools highlight cultural events and experiences so more of us can discover, share and participate in them.
+</p>
 
-## Directory Structure
+<p>
+<b>why the name?</b>
+Artists constantly evolve, inventing new genres and redefining their methods. The processes and mediums of the past aren’t always sufficient for the challenges of the future. We can thank artists for an infinity of possibilities. Production trends organize beneath the umbrella term "industry," creating an infinity of industries and our special name ;)
+</p>
+<p>Our efforts are currently focused on Lexington, Kentucky USA with lots of future plans for world domination.</p>
 
+## Project Components
+The project is split into API and various components that source information from it. We are intentionally trying to centralize the information and de-centralize the types of services that consume the data.
+### API
+Our db and a RESTful interface to our db.
+
+[API Documentation](https://github.com/infinite-industries/infinite/tree/master/api-server)
+### Web Portal
+Most of the world sees us here.
```

Comment:
We could potentially call this out a bit more explicitly:

The frontend web application served at [infinite.industries](https://infinite.industries). This is how most of the world sees us.